src/index.js

```diff
@@ -209,6 +209,12 @@ function debugCommand(config, {_chalk = chalk, _process = process, _console = co
           bar.increment(1);
         }
       });
+
+      // Make sure we are setting proper plural forms. This fixes various
+      // issues with gnu gettext. See
+      // https://github.com/mozilla/potools/issues/7 for more details.
+      po.headers['Plural-Forms'] = 'nplurals=2; plural=(n != 1);';
```

Comment:
This will obliterate what is supplied via the pot file. As mentioned in the issue comments addons-frontend doesn't have the same problem because our pot file has plural forms set.